specviz/widgets/plotting.py

```diff
@@ -297,21 +260,40 @@ def spectral_axis_unit(self, value):
 
     @property
     def selected_region(self):
-        """
-        The currently selected region object.
-        """
+        """Returns currently selected region object."""
         return self._selected_region
 
     @property
     def selected_region_bounds(self):
         """
-        The bounds of the currently selected region as a tuple of
-        `~astropy.units.Quantity` objects.
+        Returns the bounds of the currently selected region as a tuple of
+        quantities.
         """
         if self.selected_region is not None:
             return self.selected_region.getRegion() * u.Unit(
                 self.spectral_axis_unit or "")
 
+    @property
+    def region_mask(self):
```

Comment:
This method should be remove as it's not used anymore.

Reply:
Done.